nixos/tests/mpd.nix

```
@@ -1,12 +1,13 @@
-import ./make-test-python.nix ({ pkgs, lib, ... }:
+import ./make-test-python.nix (
+  { pkgs, lib, ... }:
   let
     track = pkgs.fetchurl {
       # Sourced from http://freemusicarchive.org/music/Blue_Wave_Theory/Surf_Music_Month_Challenge/Skyhawk_Beach_fade_in
-      # License: http://creativecommons.org/licenses/by-sa/4.0/
 
       name = "Blue_Wave_Theory-Skyhawk_Beach.mp3";
       url = "https://freemusicarchive.org/file/music/ccCommunity/Blue_Wave_Theory/Surf_Music_Month_Challenge/Blue_Wave_Theory_-_04_-_Skyhawk_Beach.mp3";
-      sha256 = "0xw417bxkx4gqqy139bb21yldi37xx8xjfxrwaqa0gyw19dl6mgp";
+      hash = "sha256-91VDWwrcP6Cw4rk72VHvZ8RGfRBrpRE8xo/02dcJhHc=";
+      meta.license = lib.licenses.cc-by-nc-sa-40;
```

http://freemusicarchive.org/music/Blue_Wave_Theory/Surf_Music_Month_Challenge/Skyhawk_Beach_fade_in specifies that the license is CC BY SA 4.0. Setting this to CC BY NC SA 4.0 makes the test closure unfree

```suggestion
      meta.license = lib.licenses.cc-by-sa-40;
```